src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+---
+title: ChunksWebpackPlugin
+contributors:
+  - yoriiis
+related:
+  - title: SplitChunksPlugin
+    url: /plugins/split-chunks-plugin
+  - title: The real power of Webpack 4 SplitChunks Plugin
+    url: https://medium.com/@Yoriiis/the-real-power-of-webpack-4-splitchunks-plugin-fad097c45ba0
+---
+
+The ChunksWebpackPlugin creates HTML files with entry points and chunks relations to serve your webpack bundles. It is suitable with multiple page application which contains multiple entry points.
+
+Since Webpack 4, SplitChunksPlugin offer the possibility to optimizes all chunks. It can be particularly powerful, because it means that chunks can be shared even between async and non-async chunks. See the webpack documentation for [`splitChunks.chunks`](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkschunks).
+
+This option automatically generate new chunks associated with an entry point. For example, entry points `a.js` and `b.js` share common codes with the file `vendors~a~b.js`.
+
+With multiple entry points, it can be difficult to identify relation between auto-generated chunks and entry points.
+
+The plugin parse the `entrypoints` [Map](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Map) object from the webpack compilation to get all valid entry points and associated files. Then, it generates HTML files which include all assets filtered by entry point and the `chunks-manifest.json` file.
+
+## Zero config
+
+The plugin works without configuration. For advanced usage, see the section [using configuration](#using-a-configuration).
+
+## Installation
+
+The plugin is available as a package with the name of `chunks-webpack-plugin` on [npm](https://www.npmjs.com/package/chunks-webpack-plugin) and [Github](https://github.com/yoriiis/chunks-webpack-plugin).
+
+```bash
+npm install chunks-webpack-plugin --save-dev
+```
+
+```bash
+yarn add chunks-webpack-plugin --dev
+```
+
+## Environment
+
+ChunksWebpackPlugin was built for Node.js `>=8.11.2` and webpack `>=4.x`.
+
+## Example
+
+For example implementation, please see the [Github documentation](https://github.com/yoriiis/chunks-webpack-plugin/blob/master/README.md#example).
+
+## Basic usage
+
+The plugin will generate two HTML files for each entry points. Each filename contains the entry point name, the `{{entry}}` placeholder is automatically replaced.
+
+- `{{entry}}-styles.html`: contains all HTML `<link>` tags
+- `{{entry}}-scripts.html`: contains all HTML `<script>` tags
+
+First, let's add the plugin to the webpack configuration.
+
+```js
+var ChunksWebpackPlugin = require('chunks-webpack-plugin');
```

Comment:
```suggestion
const ChunksWebpackPlugin = require('chunks-webpack-plugin');
```

Reply:
Fix in this [commit](https://github.com/webpack/webpack.js.org/pull/3771/commits/db69028c10ceaae65d296b3d28412c5f13fc154e#diff-d8ef1796282805fac08fc65ac89c9633R56)